core/common/src/main/java/alluxio/wire/MasterInfo.java

```diff
@@ -18,48 +18,158 @@
 import javax.annotation.concurrent.NotThreadSafe;
 
 /**
- * Information about the Alluxio master.
+ * The Alluxio master information.
```

Comment:
Add a comment to about how this info is shared b/w primary / secondary masters and how to distinguish

Reply:
The master info is only the information about the master that is currently running and controlling the Alluxio cluster.
changed to:
/**
 * The information of the Alluxio leader master.
 */